client/packages/common/src/intl/locales/en/common.json

```
@@ -601,7 +601,7 @@
   "table.show-columns": "Show / hide columns",
   "title.confirm-delete-encounter": "Delete encounter",
   "warning.caps-lock": "Warning: Caps lock is on",
-  "description.notification-preferences": "By default, the system will tell you if you have more than 3 or 6 months of stock on the homepage dashboard.",
+  "description.notification-preferences": "By default, the system will tell you if you have less than 3 or 6 months of stock on the homepage dashboard.",
```

```suggestion
  "description.notification-preferences": "By default, the system will tell you if you have less than 3 or more than 6 months of stock on the homepage dashboard.",
```

Is it too wordy though? Is this ugly though?

`"By default, the system will tell you if you have <3 or >6 months of stock on the homepage dashboard.`

---

Think it's fine to be wordy... looks weird with < 3 or > 6